### PR TITLE
OKTA-432207: deprecating org logo API

### PR DESCRIPTION
**Pull request by arsalanbadar-okta**
## Description:
- **What's changed?** deprecating org logo API
- **Is this PR related to a Monolith release?** 2021.12.0

### Resolves:

* [OKTA-432207](https://oktainc.atlassian.net/browse/OKTA-432207)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/org/index.md [hunk 1]
```diff
@@ -393,10 +393,12 @@ The Org Logo API has the following CRUD operations:
 
 ### Upload Logo for org
 
-<ApiOperation method="post" url="/api/v1/org/logo" />
+<ApiOperation method="post" url="/api/v1/org/logo" /> <ApiLifecycle access="deprecated" />
 
 Updates the logo for your org
 
+>**Note:** This endpoint is deprecated. Use the [Upload Theme Logo](/docs/references/api/brands/index.md#upload-theme-logo) API instead.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/org/index.md:400]:
```suggestion
>**Note:** This endpoint is deprecated. Use the [Upload Theme logo](/docs/references/api/brands/index.md#upload-theme-logo) endpoint instead.
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM

One small suggested change.

**Review by albertchen-okta — APPROVED** [review]
lgtm

**Comment by albertchen-okta** [comment]
> LGTM
> 
> One small suggested change.

fyi I just noticed another deprecated API refers to "API" not "Endpoint" - no matter to me, just happened to see the difference:
Screen Shot 2021-11-18 at 2 46 42 PM

**Comment by brentschaus-okta** [comment]
> > LGTM
> > One small suggested change.
> 
> fyi @brentschaus-okta I just noticed another deprecated API refers to "API" not "Endpoint" - no matter to me, just happened to see the difference: Screen Shot 2021-11-18 at 2 46 42 PM
> 
> I think your way might be more accurate? Otherwise its "use this Brands API instead"

That was my thinking. I used our approach in the Release Notes as the model. For [example](https://developer.okta.com/docs/release-notes/2021-okta-identity-engine/#new-devices-api-response-property-available).

**Pull request merged**